hw1103/src/main/java/com/example/hw1103/CompoundView.java

```diff
@@ -0,0 +1,64 @@
+package com.example.hw1103;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.DrawableRes;
+import android.support.annotation.Nullable;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.graphics.drawable.DrawableCompat;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.example.hw1103.backend.User;
+
+public class CompoundView extends LinearLayout {
+
+    private ImageView icon;
+    private TextView name;
+    private TextView email;
+
+    public CompoundView(final Context context) {
+        super(context);
+        initCompoundView();
```

Comment:
Instead of call initCompoundView in all constructors you can call construstor with more params fro construtor with less parasm. Like 
public CompoundView(final Context context) {
this(context, null);

Reply:
done